### PR TITLE
GUVNOR-2431: Redesign Copy, Save, Rename, Restore and Delete Popups (away from FormStylePopup)

### PR DESCRIPTION
**Pull request by karreiro**
This PR updates the usage of PopUps in order to call the new PopUps API (that implements the MVP pattern using Errai UI).

See:
- https://issues.jboss.org/browse/GUVNOR-2431
- https://github.com/uberfire/uberfire/pull/443

### CODE REVIEW

File: drools-wb-screens/drools-wb-guided-rule-editor/drools-wb-guided-rule-editor-client/src/main/java/org/drools/workbench/screens/guided/rule/client/editor/GuidedRuleEditorPresenter.java [hunk 1]
```diff
@@ -206,7 +206,7 @@ protected void save() {
                                                                              GuidedRuleEditorResources.CONSTANTS );
 
         if ( validator.isValid() ) {
-            saveOperationService.save( versionRecordManager.getPathToLatest(),
+            savePopUpPresenter.show( versionRecordManager.getPathToLatest(),
```
Comment (manstis) [inline drools-wb-screens/drools-wb-guided-rule-editor/drools-wb-guided-rule-editor-client/src/main/java/org/drools/workbench/screens/guided/rule/client/editor/GuidedRuleEditorPresenter.java]: @karreiro Could you please run the [KIE Code Style](https://github.com/droolsjbpm/droolsjbpm-build-bootstrap/tree/master/ide-configuration/intellij-configuration/code-style) over this class? It's a good habit to get into to format code before submitting.

**Comment by manstis** [comment]
Changes are OK, but I'd like the code formatted before merging.. sorry.

**Comment by karreiro** [comment]
@manstis Thanks :)

**Comment by karreiro** [comment]
Jenkins please retest this.

**Comment by manstis** [comment]
Thanks @karreiro +1 to merge (with others when green).

**Comment by karreiro** [comment]
Jenkins please retest this.

**Comment by karreiro** [comment]
Jenkins please retest this.

**Comment by psiroky** [comment]
Just FYI: in case there is a compilation error in the PR build (and not test failure) it is highly unlikely that it is an environment related issue/random failure, so re-running the build will likely not help. There are couple of things to try: make sure all the PRs are up to date (e.g. rebase on top of the latest target branch changes) --- and of course make sure the different PR branch names do match.

**Comment by karreiro** [comment]
@psiroky. I had exactly that rebase problem :( unfortunately I took sometime to notice. Thanks :)

**Comment by manstis** [comment]
+1 to merge with others.

**Pull request merged**